src/main/java/types/HostCpuPinning.java

```diff
@@ -0,0 +1,58 @@
+package types;
+
+import org.ovirt.api.metamodel.annotations.Link;
+import org.ovirt.api.metamodel.annotations.Type;
+
+/**
+ * Type representing a physical CPU of a host with
+ * the current pinning status.
+ *
+ * @author Lucia Jelinkova <ljelinko@redhat.com>
+ * @since 4.5
+ * @date 15 Feb 2022
+ * @status added
+ */
+@Type
+public interface HostCpuPinning extends Identified {
```

Comment:
how about adding what cpu(s) is used for VDSM? we should have this information on the engine side now

Reply:
Do you think a simple boolean would be enough? What about `runsVdsm` name?